AI/scripts/upload_to_kaggle.py

```
@@ -0,0 +1,151 @@
+    sys.exit(1)
+
+print(f">> 업로드할 parquet 파일: {len(parquet_files)}개")
+for f in parquet_files:
+    fpath = os.path.join(OUTPUT_DIR, f)
+    size  = os.path.getsize(fpath) / (1024 * 1024)
+    print(f"   - {f} ({size:.1f} MB)")
+
+# ─────────────────────────────────────────────────────────────────────────────
+# Kaggle 데이터셋 버전 업데이트
+# ─────────────────────────────────────────────────────────────────────────────
+print(f"\n>> Kaggle 데이터셋 업로드 중...")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)